.github/workflows/fpga.yml

```diff
@@ -249,7 +249,11 @@ jobs:
                        package(caliptra-builder) |
                        package(caliptra-cfi-derive) |
                        package(caliptra-file-header-fix) |
-                       package(compliance-test))'
+                       package(compliance-test) |
```

Comment:
I think we should add a comment that includes a link to the tracking issue to remove this filter change